### ISSUE TITLE
Cannot have two HTML5 backends at the same time.

### ISSUE THREAD

**Issue by szopenkrk** [adazzle/react-data-grid#863]
Hi we have problem witch react-data-grid and react-big-calendar which use react-dnd and react-dnd-html5-backend. After create component react-data-grid and open component react-big-calendar application crashed because of Cannot have two HTML5 backends at the same time.
How solve that ? Its important for us 

**WHICH VERSION OF REACT ARE YOU USING?**
```
Officially Supported:
[x] v0.14.x
```

```
Community Supported:
[ ] v15.0.x
```

**WHICH BROWSER ARE YOU USING?**

```
Officially Supported:
[ ] IE 9 / IE 10 / IE 11
[ ] Edge
[x] Chrome
```

```
Should work:
[ ] Firefox
[ ] Safari
```
--------

**I'm submitting a ...**  (check one with "x")
```
[x ] bug report
[ ] feature request
[ x] support request => Please do not submit support request here, instead see https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md
```

**Current behavior**
Show error Cannot have two HTML5 backends at the same time.

**Expected/desired behavior**
Should open properly component on the same instance of dnd

**Comment by ejnu** [comment]
Please provide passing own html5 backend like it is in big-calendar-data

https://github.com/intljusticemission/react-big-calendar/issues/460

**Comment by lord-xeon** [comment]
I just recently came into this issue as well and it's turning into a blocking issue for me very quickly.

@tquetano-r7 I've tried to locally apply those changes to the DraggableContainer.js file:

`react-data-grid/packages/react-data-grid-addons/src/draggable-header/DraggableContainer.js`

I'm still getting the error thrown from `react-dnd-html5-backend` about multiple backends.  I've already spent a full day investigating it, and I fear that more time will be sunk into it unless someone can help with a temporary workaround/hack/fix.

Thanks!

**Comment by lord-xeon** [comment]
I did manage to get my grid working by using the code provided by @tquetano-r7.  I had to modify [DraggableContainer.js file](https://github.com/adazzle/react-data-grid/blob/master/packages/react-data-grid-addons/src/draggable-header/DraggableContainer.js):

`react-data-grid/packages/react-data-grid-addons/src/draggable-header/DraggableContainer.js`

As well as my own code.  In addition, I had to move up where I originally put the `
<DragDropContextProvider />` in my code, I think redux was messing with it, which was causing my error.

It should be noted that I needed to add

```javascript
static contextTypes = {
    dragDropManager: PropTypes.object
};

getChildContext() {
  const dragDropManager = this.context.dragDropManager || getDefaultDragDropManager();

  return {
    dragDropManager
  };
}
```

To the parent component in every place where I would have done `@DragDropManager(HTML5Backend)`
